src/main/scala/com/tuplejump/snackfs/cassandra/loadbalancing/LocalMachineRoundRobinPolicy.scala

```diff
@@ -0,0 +1,46 @@
+package com.tuplejump.snackfs.cassandra.loadbalancing

```

Comment:
This policy can be provided in the application. In the core system we should rely on default Cassandra policies as long as they don't break anything.